### PR TITLE
Android ci fix

### PR DESCRIPTION
**Pull request by KitHat**

### CODE REVIEW

File: libindy/android.test.sh [hunk 1]
```diff
@@ -38,12 +38,10 @@ build_test_artifacts(){
...
 
         # TODO empty for full testing SET_OF_TESTS=''
-        SET_OF_TESTS='--lib --test interaction'
+        SET_OF_TESTS='--test interaction'
 
-        # build - separate step to see origin build output
-        # TODO move RUSTFLAGS to cargo config and do not duplicate it here
         RUSTFLAGS="-L${TOOLCHAIN_DIR}/sysroot/usr/${TOOLCHAIN_SYSROOT_LIB} -lc -lz -L${TOOLCHAIN_DIR}/${TRIPLET}/lib -L${LIBZMQ_LIB_DIR} -L${SODIUM_LIB_DIR} -lsodium -lzmq -lgnustl_shared" \
-                    cargo test ${BUILD_TYPE} --target=${TRIPLET} ${SET_OF_TESTS} --no-run
+            cargo build --release --target=${TRIPLET}
```
Comment (jovfer) [inline libindy/android.test.sh]: please use variables instead of fixed parameters where it's possible

File: libindy/android.test.sh [hunk 2]
```diff
@@ -73,6 +71,9 @@ execute_on_device(){
     adb -e push \
     "${LIBZMQ_LIB_DIR}/libzmq.so" "/data/local/tmp/libzmq.so"
 
+    adb -e push \
```
Comment (jovfer) [inline libindy/android.test.sh]: is this line required because we drop unit tests from the CI scope?

File: libindy/android.test.sh (hunk 1 above)
Comment (KitHat) [reply libindy/android.test.sh]: Changed already

File: libindy/android.test.sh (hunk 2 above)
Comment (KitHat) [reply libindy/android.test.sh]: Yep. Libindy is not compiled into any more.

File: libindy/android.test.sh [hunk 3]
```diff
@@ -40,10 +40,8 @@ build_test_artifacts(){
         # TODO empty for full testing SET_OF_TESTS=''
         SET_OF_TESTS='--lib --test interaction'
 
-        # build - separate step to see origin build output
-        # TODO move RUSTFLAGS to cargo config and do not duplicate it here
         RUSTFLAGS="-L${TOOLCHAIN_DIR}/sysroot/usr/${TOOLCHAIN_SYSROOT_LIB} -lc -lz -L${TOOLCHAIN_DIR}/${TRIPLET}/lib -L${LIBZMQ_LIB_DIR} -L${SODIUM_LIB_DIR} -lsodium -lzmq -lgnustl_shared" \
-                    cargo test ${BUILD_TYPE} --target=${TRIPLET} ${SET_OF_TESTS} --no-run
+            cargo build ${BUILD_TYPE} --target=${TRIPLET}
```
Comment (jovfer) [inline libindy/android.test.sh]: it's important to keep 2 steps: the first one for full build of tests and the second one for parsing tests to running

**Pull request merged**